### PR TITLE
Install/Dev check: Add new test cases for some modules

### PR DESCRIPTION
**Pull request by njayaram2**
Some modules such as array_ops and pmml did not have any install check
files, while stemmer did not have any test files. This commit adds some
basic test cases for these modules.

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/691/

File: methods/array_ops/src/pg_gp/test/array_ops.ic.sql_in [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+---------------------------------------------------------------------------
+-- Setup:
+---------------------------------------------------------------------------
+CREATE TABLE data AS
+SELECT
+    '{1,2,3}'::float8[] AS an,
+    '{4,5,7}'::float8[] AS b;
+
+SELECT MADLIB_SCHEMA.array_dot(
```
Comment (jingyimei) [inline methods/array_ops/src/pg_gp/test/array_ops.ic.sql_in]: For adding MADLIB_SCHEMA in front of function name: refer to the discussion in PR #325 . We need to keep them consistent.

File: methods/stemmer/src/pg_gp/test/porter_stemmer.sql_in [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ *//* ----------------------------------------------------------------------- */
+
+CREATE TABLE token_tbl ( id integer,
```
Comment (jingyimei) [inline methods/stemmer/src/pg_gp/test/porter_stemmer.sql_in]: Noticed the install check and dev check test cases are exactly the same as each other. Is it better to differentiate them?

File: src/ports/postgres/modules/pmml/test/pmml.ic.sql_in [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+SELECT MADLIB_SCHEMA.glm(
+    'abalone',
+    'abalone_logit_out',
+    'rings < 10',
+    'ARRAY[1, length, diameter, height, whole, shucked, viscera, shell]',
+    'family=binomial, link=logit', NULL, 'max_iter=1000, tolerance=1e-16'
+);
+
+
+SELECT MADLIB_SCHEMA.pmml('abalone_probit_out');
```
Comment (jingyimei) [inline src/ports/postgres/modules/pmml/test/pmml.ic.sql_in]: I just noticed both in IC and DC we only test pmml with one argument. Should we add a test case for the second argument 'name_spec' in dev check?

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/692/

**Pull request closed without merging**